giraffe/src/components/SizedTable.tsx

```diff
@@ -113,22 +105,6 @@ export const SizedTable: FunctionComponent<Props> = ({
                   </FluxTablesTransform>
                 )
 
-              case LayerTypes.Custom:
-                const renderProps = {
-                  key: layerIndex,
-                  width,
-                  height,
-                  innerWidth: env.innerWidth,
-                  innerHeight: env.innerHeight,
-                  xScale: env.xScale,
-                  yScale: env.yScale,
-                  xDomain: env.xDomain,
-                  yDomain: env.yDomain,
-                  columnFormatter: env.getFormatterForColumn,
-                  yColumnType: env.yColumnType,
-                }
-                return layerConfig.render(renderProps)
-
```

Comment:
`SizedTable` no longer can have a custom layer. This change makes sense because putting a canvas on top of non-canvas components is asking for bugs.

Custom layers will be limited to being in `SizedPlot` only.